packages/prisma-client/migrations-cli/commands.ts

```diff
@@ -1,21 +1,50 @@
 import { FileLocker, MigrationMeta } from "umzug";
 import fs from "node:fs";
 import path from "node:path";
+import * as readline from "node:readline";
 import * as prismaMigrations from "./prisma-migrations";
 import { umzug } from "./umzug";
 import * as logger from "./logger";
 import args from "./args";
+import { UserError } from "./errors";
 
 const templateFilePath = path.join(
   prismaMigrations.migrationsDir,
   "template.txt"
 );
 const lockfilePath = path.join(prismaMigrations.migrationsDir, "lockfile");
 
-const failWith = (message: string) => {
-  logger.error(message);
-  logger.error("");
-  process.exit(1);
+const confirm = async (
```

Comment:
there are higher level libs that allow you to do prompts in 1 loc

Reply:
I'm hesitant about adding dependencies usually. I worry about node_modules bloat and security.

https://twitter.com/DennisCode/status/1474972713861091329

Reply:
I am sure there are really nice well-made packages, I can recommend one I used if you want

Reply:
I don't feel a need to use a dependency here to be honest

Reply:
I say it not only for the sake of reducing this line, but so that we have our set of dependencies we trust to build clis, this is not the only one we have, there is already one in https://github.com/webstudio-is/webstudio-cli

and will have more of them in the future ... makes sense when you look at it from global perspective?